src/org/exist/storage/BrokerPool.java

```diff
@@ -871,6 +870,16 @@ protected void initialize() throws EXistException, DatabaseConfigurationExceptio
                         symbols = new SymbolTable(conf);
                         isReadOnly = isReadOnly || !symbols.getFile().canWrite();
 
+                        try {
+                            // initialize EXPath repository so indexManager and
+                            // startup triggers can access it
+                            expathRepo = ExistRepository.getRepository(this.conf);
+                        } catch(final PackageException e) {
+                            LOG.warn("Failed to initialize expath repository: " + e.getMessage() + " - " +
```

Comment:
No, don't do that! To recover your data in case of an emergency you need the broker pool to come up, even if there are inconsistencies.

A damaged expath repo is a minor issue and all your data is likely still intact, so you should be able to start up, run an export and repair afterwards.